src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -0,0 +1,278 @@
+            data-cy="ErrorMessage"
+          >
+            Name is required
+          </p>
+        )}
+      </div>
+
+      <div
+        className="field"
+        data-cy="EmailField"
+      >
+        <label
+          className="label"
+          htmlFor="comment-author-email"
+        >
+          Author Email
+        </label>
+
+        <div
+          className="
+            control
+            has-icons-left
+            has-icons-right"
+        >
+          <input
+            type="email"
+            name="email"
+            id="comment-author-email"
+            placeholder="email@test.com"
+            className={classNames(
+              'input',
+              { 'is-danger': handleNotification(email.length) },
+            )}
+            value={email}
+            onChange={(event) => setEmail(event.target.value)}
+          />
+
+          <span
+            className="
+              icon
+              is-small
+              is-left"
+          >
+            <i className="fas fa-envelope" />
+          </span>
+
+          {handleNotification(email.length) && (
+            <span
+              className="icon
+                is-small
+                is-right
+                has-text-danger"
+              data-cy="ErrorIcon"
+            >
+              <i className="fas fa-exclamation-triangle" />
+            </span>
+          )}
+        </div>
+
+        {handleNotification(email.length) && (
+          <p
+            className="help is-danger"
+            data-cy="ErrorMessage"
+          >
+            Email is required
+          </p>
+        )}
+      </div>
+
+      <div
+        className="field"
+        data-cy="BodyField"
+      >
+        <label
+          className="label"
+          htmlFor="comment-body"
+        >
+          Comment Text
+        </label>
+
+        <div className="control">
+          <textarea
+            id="comment-body"
+            name="body"
+            placeholder="Type comment here"
+            className={classNames(
+              'textarea',
+              { 'is-danger': handleNotification(body.length) },
+            )}
+            value={body}
+            onChange={(event) => setBody(event.target.value)}
+          />
+        </div>
+
+        {handleNotification(body.length) && (
+          <p
+            className="help is-danger"
+            data-cy="ErrorMessage"
+          >
+            Enter some text
+          </p>
+        )}
+      </div>
+
+      <div className="field is-grouped">
+        <div className="control">
+          <button
+            type="submit"
+            className={classNames(
+              'button',
+              'is-link',
+              { 'is-loading': isLoading },
+            )}
+            onClick={() => handleFormValidation()}
+          >
+            Add
+          </button>
+        </div>
+
+        <div className="control">
+          {/* eslint-disable-next-line react/button-has-type */}
```

Comment:
Is it causing trouble? Because it looks valid. It is better not to disable eslint, but try to fix it's problem.